recipes/graphtyper/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set version = "2.0" %}
+
+package:
+  name: graphtyper
+  version: '{{ version }}'
+
+source:
+  - url: https://github.com/DecodeGenetics/graphtyper/releases/download/v{{ version }}/graphtyper  # [linux]
```

Comment:
Can we just compile it? We can use a `git_url` and `git_rev` so we pull in the submodules.

Reply:
> Can we just compile it? We can use a `git_url` and `git_rev` so we pull in the submodules.

Sure, I can give it a shot.